src/app/routes/features/features.component.ts

```diff
@@ -15,21 +15,55 @@ import {FeatureRendererComponent} from './feature-renderer.component';
 export class FeaturesComponent implements OnInit {
 
   features: Feature[];
+  filter: string;
   gridApi: GridApi;
   columnApi: ColumnApi;
   gridOptions: GridOptions;
 
   constructor(private featureService: FeatureService, private logger: NGXLogger) {
+      const colDefs: ColDef[] = [
+          {
+              field: 'uid',
+              getQuickFilterText: (params) => {
+                  return params.value;
+              }
+          },
+          {
+              field: 'description',
+              getQuickFilterText:  (params) => {
+                  return params.value;
+              }
+          },
+          {
+              field: 'group',
+              getQuickFilterText:  (params) => {
+                  return params.value;
+              }
+          },
+          {
+              field: 'permissions',
+              getQuickFilterText:  (params) => {
+                  return params.value;
```

Comment:
Permissions are array. Are you sure you will be able to filter it by `params.value` ??